.github/workflows/post-commit.yml

```diff
@@ -224,7 +224,7 @@ jobs:
         continue-on-error: true
   integration:
     runs-on: ubuntu-18.04
-    timeout-minutes: 120
+    timeout-minutes: 150
```

Comment:
These integration tests include some HA/decommission related tests that can take up to 6-10 mins for each test. The run time varies from 90-130m in previous runs. I bumped to 150m to avoid test execution being killed due to timeout. But the change here does not increase the execution time of these tests.